packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/create-oauth-app/index.md

```diff
@@ -5,21 +5,18 @@ Create the client application that you want to use with the Okta APIs.
 
 1. Sign in to your Okta organization as a user with administrative privileges. [Create an org for free](https://developer.okta.com/signup).
 
-1. Select **Applications** and then **Applications** from the side navigation.
+1. In the Admin Console, go to **Applications** > **Applications**.
+1. Click **Create App Integration**.
+1. On the Create a new app integration page, select **OIDC - OpenID Connect** as the **Sign-in method**. Choose any **Application type**. Creating a web, single-page, or native app is an easy way to test scope-based access to Okta's APIs using an OAuth 2.0 bearer token. Click **Next**.
 
-1. On the **Applications** page, click **Add Application**.
-
-1. On the **Create New Application** page, select any app type and then click **Next**. We suggest creating a web, single-page, or native app for an easy way to test scope-based access to Okta's APIs using an OAuth 2.0 bearer token.
   > **Note:** It is important to choose the appropriate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, hence breaking the sign-in or sign-out flow.
```

Comment:
```suggestion
  > **Note:** It is important to choose the appropriate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, and would break the sign-in or sign-out flow.
```